src/gfn/gflownet/base.py

```diff
@@ -32,41 +35,41 @@ class GFlowNet(ABC, nn.Module, Generic[TrainingSampleType]):
     def sample_trajectories(
         self,
         env: Env,
-        n_samples: int,
```

Comment:
it looks like you're handling the conditioning input to this function as a kwarg, whereas `sampler`'s `sample_trajectories` have an explicit `conditioning` input. I'm wondering if you have a particular reason for this choice

Reply:
I think maybe all functions should use an explicit `conditioning` kwarg, what do you think?

Reply:
I agree that it would be cleaner

Reply:
it should be done now